_episodes/06-style-guide.md

```diff
@@ -3,13 +3,23 @@ title: "Style Guide"
 teaching: 10
 exercises: 0
 questions:
+- "What style should be used for Carpentries lessons?"
 - "How are keyboard key combinations written?"
+- "How are short spans of code written?"
 objectives:
 - "Explain keyboard key combination."
+- "Locate documentation for general matters of style in The Carpentries."
+- "Identify uses of the span of code Markdown feature."
 keypoints:
-- "Keyboard keys need to use `<kbd>` HTML tag."
+- "The Carpentries Handbook contains a style guide."
+- "Keyboard actions are described using the `<kbd>` HTML tag."
+- "The backtick (_\`_) is used as a span of code indicator to enclode code, filenames, and 
```

Comment:
enclode -> enclose

Reply:
clearly, i should stop trying to spell words working in browser, it's not going well for me...